tests/data/cpu_template_helper/fingerprint_ARM_NEOVERSE_N1_4.14host.json

```diff
@@ -156,7 +156,7 @@
       },
       {
         "addr": "0x6030000000100040",
-        "bitmap": "0b00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000010000000000000000000"
+        "bitmap": "0b00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000"
```

Comment:
Why don't we need to update baseline for Arm Neoverse-V1 + kernel 4.14?

Reply:
Because Neoverse V1 + 4.14 uses the old kernel still. Recreating that kernel will be a follow-up.